.pipelines/templates/.builder-release-template.yaml

```diff
@@ -254,12 +271,6 @@ steps:
       artifactName: 'vhd-image-bom-${{ parameters.artifactName }}'
       targetPath: 'image-bom.json'
     displayName: publish container image list
-  - task: PublishPipelineArtifact@1
```

Comment:
how come we don't need this anymore ?

Reply:
Originally, I was publishing the artifact so that it could later be consumed by some service that would evaluate the data and determine if it was within spec. I have since moved the approach to utilizing kusto using the https://github.com/Azure/azure-kusto-go sdk, which allows ingestion from the agent vm during the pipeline. The artifact so far hasn't been used at all as it was just there until I finished the next part of the project. @djsly